keps/sig-api-machinery/00xx-admission-webhooks-to-ga.md

```diff
@@ -107,8 +107,8 @@ object will be used otherwise (e.g. `pods/proxy`) the labels on the existing par
 object will be used. If subresource does not have a parent object, the Webhook call 
 may fail or skipped based on the failure policy.
 
-When deleting a collection, the webhook will get a filtered list of objects if the 
-ObjectSelector is applied. The webhook may be called multiple times with sub-lists.
+When deleting a collection, the webhook gets called for each object matching
+the ObjectSelector.
```

Comment:
Might we change this in the future? Or is this a permanent change? (I don't care either way, I just want to know :) )

Reply:
We can't change it until v2, because a change will break existing webhooks.

Reply:
Actually we don't need v2, we can add a field to the webhook configuration API to indicate if the webhook expects a list or a single object. Updated the KEP.